protocol/java/org/openyolo/protocol/CredentialRetrieveResult.java

```diff
@@ -251,10 +251,11 @@ public Builder setCredential(@Nullable Credential credential) {
          */
         private Builder setCredentialFromProto(@Nullable Protobufs.Credential credential)
                 throws MalformedDataException {
-            if (!Protobufs.Credential.getDefaultInstance().equals(credential)) {
-                mCredential = Credential.fromProtobuf(credential);
-            } else {
+            if (null == credential
```

Comment:
null check likely unnecessary, as the equality check on the default instance will return false.